### ISSUE TITLE
Snackbar only opens on the first call, does now show on any consecutive call

### ISSUE THREAD

**Issue by manasa884** [StratoDem/sd-material-ui#85]
I am trying to have a snackbar show every time a button is pressed. It shows on the first click but does not show on any clicks after that unless I quit my program and restart it. 

I used the example code from the usage file:
```
import sd_material_ui
import dash
import flask
import dash_html_components as html
import time

app = dash.Dash('')

app.scripts.config.serve_locally = True

spacer = html.Div(children=[], style=dict(height=20))
final_spacer = html.Div(children=[], style=dict(height=400))

app.layout = html.Div([
    sd_material_ui.RaisedButton(id='input10', label='Marco'),
    sd_material_ui.Snackbar(id='snackbar', open=False, message='Polo', action='Reveal')
])

# Callback for SDSnackbar
@app.callback(
    dash.dependencies.Output('snackbar', 'open'),
    [dash.dependencies.Input('input10', 'n_clicks')])
def open_snackbar(button_click):
    if button_click is not None and button_click > 0:
        return True
    else:
        return False


if __name__ == '__main__':
    app.run_server(host='127.0.0.1')
```

**Comment by mjclawar** [comment]
@manasa884 Can you please provide a small example code that causes this behavior?

**Comment by manasa884** [comment]
Edited my original post with the code I used @mjclawar

**Comment by mjclawar** [comment]
Thank you very much. 

@coralvanda can you try to replicate this/identify the issue, please? Guessing that there is some update logic that does not pass through a new `open` prop to the underlying `material-ui` `Snackbar` component

**Comment by mjclawar** [comment]
Patched in `sd-material-ui==2.16.4` (#86). Thank you for flagging this, @manasa884 !
